packages/typescript/internal/ts_project.bzl

```diff
@@ -568,6 +576,18 @@ def ts_project_macro(
             will be called with the remaining arguments.
             See the packages/typescript/test/ts_project/swc directory for an example.
 
+            When a custom transpiler is used (i.e. `transpiler != "tsc"`) then this macro expands to these targets:
+
+                - `[name]` - the default target is a `js_library` which can be included in the `deps` of downstream rules.
+                    Note that it will successfully build *even if there are typecheck failures* because the `tsc` binary
+                    is not needed to produce the default outputs.
+                    This is considered a feature, as it allows you to have a faster development mode where typechecking
+                    is not on the critical path.
+                - `[name].check` - this target will fail to build if the typechecking fails, useful for CI.
```

Comment:
How about `[name]_typings` to describe what you're building in the target. If it fails to build typings then you've got a typings error?

Reply:
Oh. I see, the `.check` target is the filegroup so the build failure will be
```
Failed to build `//path/to:_[name]_tsc` in the current naming
```

Perhaps `_[name]_tsc` should be `[name]_typings` in that case so the error message is sound?